packages/origin-247-certificate/test/setup.ts

```diff
@@ -123,11 +103,19 @@ export const bootstrapTestInstance: any = async () => {
     const certificateCommandRepository = await app.resolve<CertificateCommandRepository>(
         CERTIFICATE_COMMAND_REPOSITORY
     );
+    const blockchainPropertiesService = await app.resolve<BlockchainPropertiesService>(
+        BlockchainPropertiesService
+    );
 
     const connection = await app.resolve<Connection>(getConnectionToken());
     const tables = connection.entityMetadatas
         .filter(
-            (e) => e.tableName !== 'issuer_blockchain_properties' && e.tableName !== 'issuer_signer'
+            (e) =>
+                ![
+                    'issuer_blockchain_properties',
+                    'issuer_signer',
```

Comment:
these two can be dropped already, right?

Reply:
right, removed